owncloudApp/src/androidTest/java/com/owncloud/android/shares/db/OCShareDaoTest.kt

```diff
@@ -260,8 +260,8 @@ class OCShareDaoTest {
         )
 
         assertThat(textShares, notNullValue())
-        assertEquals(textShares.size, 2)
-        assertEquals(textShares.get(0).name, "Text 1 link")
+        assertEquals(textShares.size, 1)
+        assertEquals(textShares.get(0).name, "Text 2 link")
```

Comment:
FYI: First value is the expected value and the second one is the actual value.

You don't have to change but take it into account for coming code